packages/components/src/dropdown/index.js

```diff
@@ -72,25 +72,19 @@ class Dropdown extends Component {
 
 		return (
 			<div className={ className } ref={ this.bindContainer }>
-				{ /**
-				   * This seemingly redundant wrapper node avoids root return
-				   * element styling impacting popover positioning.
-				   */ }
-				<div>
```

Comment:
In my testing, this div was not necessary anymore. Can you confirm @aduth?

I removed it from here and from the slots because it messes with the styling of the block toolbars.